doc/api/v0.yaml

```diff
@@ -51,7 +51,7 @@ paths:
             application/json:
               schema:
                 items:
-                  $ref: "#/components/schemas/Proposal"
+                $ref: "#/components/schemas/ProposalWithChallengeInfo"
```

Comment:
Wrong indentation level.

Reply:
Good catch. Both levels are actually somewhat valid, so I guess this is why the extension I have for auto-formatting and linting OpenAPI did not complain.